modules/php/Managers/Rules.php

```diff
@@ -62,7 +62,7 @@ public static function incrementPhaseOneDrawEndAmount($amount = 1)
    */
   public static function setNewTurnRules($player, $eventCard = null)
   {
-    $amountOfCardsToDraw = $eventCard ? $eventCard->getPhaseOneAmountOfCardsToDraw() : 2;
+    $amountOfCardsToDraw = $eventCard ? $eventCard->getPhaseOneAmountOfCardsToDraw($player) : $player->defaultCardsToDraw();
```

Comment:
I'm not sure for what purpose this is used, but probably 2 is not good constant, characters should be respected